### PR TITLE
ENT-1511 | Allowing for CORS for this service

### PR DESCRIPTION
**Pull request by christopappas**

### CODE REVIEW

File: ecommerce/settings/production.py [hunk 1]
```diff
@@ -84,3 +84,12 @@ def get_env_setting(setting):
...
 
 ENTERPRISE_API_URL = urljoin(ENTERPRISE_SERVICE_URL, 'api/v1/')
+
+CORS_ORIGIN_WHITELIST = environ.get('CORS_ORIGIN_WHITELIST', ())
+CORS_URLS_REGEX = r'{regex}'.format(regex=environ.get('CORS_URLS_REGEX', ''))
+CORS_ALLOW_HEADERS = corsheaders_default_headers + (
+    'use-jwt-cookie',
+)
+CORS_ALLOW_CREDENTIALS = True
+CORS_ORIGIN_ALLOW_ALL = environ.get('CORS_ORIGIN_ALLOW_ALL', False)
```
Comment (georgebabey) [inline ecommerce/settings/production.py]: Is this and `CORS_ALLOW_INSECURE ` needed at all? I can't imagine a scenario where we'd be using production.py and want to allow any origin.

**Review by georgebabey — APPROVED** [review]
Just one question but this looks good!

File: ecommerce/settings/production.py (hunk 1 above)
Comment (christopappas) [reply ecommerce/settings/production.py]: can remove, np

File: ecommerce/settings/production.py [hunk 2]
```diff
@@ -84,3 +84,10 @@ def get_env_setting(setting):
 # END PAYMENT PROCESSOR OVERRIDES
 
 ENTERPRISE_API_URL = urljoin(ENTERPRISE_SERVICE_URL, 'api/v1/')
+
+CORS_ORIGIN_WHITELIST = environ.get('CORS_ORIGIN_WHITELIST', ())
+CORS_URLS_REGEX = r'{regex}'.format(regex=environ.get('CORS_URLS_REGEX', ''))
+CORS_ALLOW_HEADERS = corsheaders_default_headers + (
```
Comment (mushtaqak) [inline ecommerce/settings/production.py:91]: We need to import this as well.
```python
from corsheaders.defaults import default_headers as corsheaders_default_headers
```

**Review by mushtaqak — CHANGES_REQUESTED** [review]
@christopappas Looks good, one thing, we need to import `corsheaders_default_headers`

File: ecommerce/settings/production.py (hunk 2 above)
Comment (christopappas) [reply ecommerce/settings/production.py:91]: goood catch. thank you mushtaq

**Review by mushtaqak — APPROVED** [review]
Looks good :shipit:

**Pull request merged**